meow/frontend/src/components/EditPost/EditForm.js

```diff
@@ -119,8 +119,26 @@ class EditForm extends React.Component {
     setFieldsValue({ tags });
   };
 
+  handleClickpub = e => {
+    this.setState({
+      ...this.state,
+      click_pubready: e.target.checked
+    });
+  };
+
   render() {
     const { getFieldDecorator, getFieldsValue } = this.props.form;
+
+    let flag_urlwarn = null;
```

Comment:
this is a really small thing but generally flags are boolean variables so this name might be confusing since it actually stores a JSX component